tests/fileVersionTest.js

```diff
@@ -158,35 +232,36 @@ describe('Main: Currently testing file versions management,', function () {
     })
 
     it('retrieves file versions', async function () {
-      const provider = createProvider(true, true)
-      await getCapabilitiesInteraction(provider)
-      await getCurrentUserInformationInteraction(provider)
-      await PropfindFileVersionOfExistentFiles(provider)
-      await getFileVersionContents(provider)
+      // TODO: move this loop to a single provider test
+      // https://github.com/pact-foundation/pact-js/issues/604
+      for (let i = 0; i < fileInfo.versions.length; i++) {
+        const provider = createProvider(true, true)
+        await getCapabilitiesInteraction(provider, config.testUser, config.testUserPassword)
+        await getCurrentUserInformationInteraction(provider, config.testUser, config.testUserPassword)
+        await PropfindFileVersionOfExistentFiles(provider)
+        await getFileVersionContents(provider, i + 1)
 
-      return provider.executeTest(async () => {
-        const oc = createOwncloud()
-        await oc.login()
-        await oc.fileVersions.listVersions(fileInfo.id).then(versions => {
-          expect(versions.length).toEqual(2)
-          expect(versions[0].getSize()).toEqual(2)
-          expect(versions[1].getSize()).toEqual(1)
-        })
-        await oc.fileVersions.getFileVersionContents(fileInfo.id, fileInfo.versions[0].versionId).then(content => {
-          expect(content).toBe(fileInfo.versions[0].content)
-        })
-        return oc.fileVersions.getFileVersionContents(fileInfo.id, fileInfo.versions[1].versionId).then(content => {
-          expect(content).toBe(fileInfo.versions[1].content)
+        return provider.executeTest(async () => {
```

Comment:
```suggestion
        await provider.executeTest(async () => {
```
else the test will only run once